File: BarbeariaEInclusao/src/app/model/servico.ts
export class Servico {
    idServico: number;
    nomeServico: string;
    descricaoServico: string;
    precoServico: number;
    fotoServico: string;
    idBarbeariaServico: number;
    idBarbeiroServico: number;

    constructor() {
        this.idServico = 0;
        this.nomeServico = '';
        this.descricaoServico = '';
        this.precoServico = 0;
        this.fotoServico = '';
        this.idBarbeariaServico = 0;
        this.idBarbeiroServico = 0;
    }
}

File: barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/src/main/java/br/cefetmg/barbeariainclusao/model/Servico.java
package br.cefetmg.barbeariainclusao.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Servico {

    private Long idServico;
    private String nomeServico;
    private String descricaoServico;
    private double precoServico;
    private long idBarbeariaServico;
    private long idBarbeiroServico;
}
